integration-tests/gradle/src/test/java/io/quarkus/gradle/ConditionalDependenciesTest.java

```diff
@@ -86,6 +86,7 @@ public void shouldImportConditionalDependency() throws IOException, URISyntaxExc
 
         final Path deploymentLib = buildDir.toPath().resolve("quarkus-app").resolve("lib").resolve("deployment");
         assertThat(deploymentLib.resolve("org.acme.transitive-dependency-1.0-SNAPSHOT.jar")).exists();
+        assertThat(deploymentLib.resolve("io.quarkus.quarkus-agroal-999-SNAPSHOT.jar")).doesNotExist();
```

Comment:
We can't rely on the `999-SNAPSHOT` version in the tests.

Reply:
nice catch, always miss this one.